packages/trader/src/App/Components/Elements/PositionsDrawer/positions-modal-card.jsx

```diff
@@ -212,8 +212,8 @@ const PositionsModalCard = ({
         <React.Fragment>
             <NavLink
                 className={classNames('dc-contract-card', {
-                    'dc-contract-card--green': !is_multiplier && profit_loss > 0 && !result,
-                    'dc-contract-card--red': !is_multiplier && profit_loss < 0 && !result,
+                    'dc-contract-card--green': profit_loss > 0 && !result,
```

Comment:
@kate-deriv should we allow it for is_multiplier now? is it related to this PR?

Reply:
@maryia-deriv the new design is about consistency in style, so Viktor asked me to added gradient to all contract cards for all trade types. I also mentioned it in the qa-checklist

Reply:
Here are notes from figma design:
_Hence, from now onwards for Desktop and Responsive ,  the contract card must reflect the contract status  for ongoing contract and expired contract.
Now, we will show the contract status (gradient) for  ongoing contract and expired contract.
This applies to desktop and responsive._